app/controllers/gears_controller.rb

```diff
@@ -0,0 +1,55 @@
+class GearsController < ApplicationController
+  before_action :set_gear, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @gear = Gear.all
+  end
+
+  def new
+    @gear = Gear.new
+  end
+
+  def create
+    @gear = Gear.new(gear_params)
+    if @gear.save?
+      redirect_to "pages#home" # needs to be changed when routes are added
+    else
+      render 'new'
+    end
+  end
+
+  def show
+  end
+
+
+  def edit
+  end
+
+  def update
+    if @gear.save?
+      redirect_to "pages#home" # needs to be changed when routes are added
+    else
+      render 'new'
```

Comment:
I would render 'edit' instead. This way the user is not presented with a blank form upon a failed save action.